packages/api/src/routes/medical/patient.ts

```diff
@@ -280,11 +280,14 @@ router.get(
     const cxId = getCxIdOrFail(req);
     const patientId = getFrom("params").orFail("id", req);
     const patient = await getPatientOrFail({ cxId, id: patientId });
-    const respPayload: QueryProgressFromSDK = {
-      status: patient.data.consolidatedQuery?.status ?? null,
+    const consolidatedQueries = patient.data.consolidatedQueries ?? null;
```

Comment:
need to update the docstring for this function too

Reply:
Wdym?